app/views/layouts/application.html.erb

```diff
@@ -12,46 +12,59 @@
   </head>
 
   <body>
-    <ul class ='nav'>
-      <li><%= link_to 'Turing :: SO', root_path %></li>
-      <li class="rightFloatNavLink"><%= link_to 'Register', new_user_registration_path %></li>
-      <li class="dropdown">
-        <a href="javascript:void(0)" class="dropbtn">Forums</a>
-        <div class="dropdown-content">
-          <%= link_to 'Technical', root_path %>
-          <%= link_to 'Professional', root_path %>
-      </li>
-      <li><%= link_to 'Ask a Question', root_path %></li>
-      <li>
-        <div id="nav-bar-search">
-          <form class="form-inline my-2 my-lg-0">
-            <input class="form-control mr-sm-2" type="text" placeholder="Search" aria-label="Search">
-            <button class="btn btn-secondary my-2 my-sm-0" type="submit">Search</button>
-          </form>
-        </div>
-      </li>
-    </ul>
-    <div class="mainContentWrapper">
+    <section class="nav">
+      <ul>
+        <li><%= link_to 'Turing :: SO', root_path %></li>
+        <li class="dropdown">
+          <a href="javascript:void(0)" class="dropbtn">Forums</a>
+          <div class="dropdown-content">
+            <%= link_to 'Technical', root_path %>
+            <%= link_to 'Professional', root_path %>
+          </div>
+        </li>
+        <li><%= link_to 'Ask a Question', root_path %></li>
+        <li>
+          <div id="nav-bar-search">
+            <form class="form-inline my-2 my-lg-0">
+              <input class="form-control mr-sm-2" type="text" placeholder="Search" aria-label="Search">
+              <button class="btn btn-secondary my-2 my-sm-0" type="submit">Search</button>
+            </form>
+          </div>
+        </li>
+        <% if current_user.nil? %>
+          <li class="dropdown" id="logInRight">
+            <a href="javascript:void(0)" class="dropbtn">Sign In :: Register</a>
+            <div class="dropdown-content">
+              <%= link_to 'Sign In', new_user_session_path %>
```

Comment:
I literally wrote the exact same line of code for my testing branch!! lol thats great